chatgpt-app/alloy-server-sdk/aepEdgeClient.js

```diff
@@ -0,0 +1,331 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/** same as ../../common/aepEdgeClient but as an ES module and inside our own monorepo */
+
+import { isNotBlank } from "@adobe/target-tools";
```

Comment:
If isNotBlank the only thing we are using from target-tools, it may be better just to create a utility, and not have the dependency.